crates/sui-indexer/src/models_v2/transactions.rs

```diff
@@ -67,6 +68,7 @@ impl From<&IndexedTransaction> for StoredTransaction {
 impl StoredTransaction {
     pub fn try_into_sui_transaction_block_response(
         self,
+        options: &SuiTransactionBlockResponseOptions,
```

Comment:
nice!
We may also want to not load the entire `StoredTransaction` but only the needed columns, but it's a bit of work (to write the code)

Reply:
Yeah i think for a first pass we can probably get away without over optimizing but we definitely may want to do that in the future